types/expectations.d.ts

```diff
@@ -0,0 +1,17 @@
+declare namespace Chai {
```

Comment:
unused?

Reply:
hmm might need it later but we are using jest in the module so ill remove it

Reply:
lets remove this

Reply:
done